woonuxt_base/app/components/DownloadableItems.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+const { formatDate } = useHelpers();
+
+const props = defineProps({
+  downloadableItems: { type: Object as PropType<DownloadableItem[]>, required: true },
+});
+
+const downloadFile = (downloadUrl: string) => {
+  window.open(downloadUrl, '_blank');
+};
+</script>
+
+<template>
+  <table class="w-full text-left table-auto" aria-label="Download List">
+    <thead>
+      <tr>
+        <th>{{ $t('messages.general.product') }}</th>
+        <th>{{ $t('messages.shop.downloadsRemaining') }}</th>
+        <th>{{ $t('messages.shop.expires') }}</th>
+        <th class="text-right">{{ $t('messages.general.download') }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="download in props.downloadableItems" :key="download.id">
+        <td class="rounded-l-lg">
+          <router-link :to="`/product/${download.product?.slug}`" class="cursor-pointer hover:underline">{{ download.product?.name }}</router-link>
+        </td>
+        <td>{{ download.downloadsRemaining ? download.downloadsRemaining : '∞' }}</td>
+        <td>{{ download.accessExpires ? formatDate(download.accessExpires) : 'Never' }}</td>
+        <td class="text-right rounded-r-lg" v-if="download.url">
+          <button @click.stop="downloadFile(download.url)" class="text-blue-500 hover:text-blue-700 hover:underline cursor-pointer">
+            {{ download.name }}
+          </button>
```

Comment:
Instead of using a button element, we can use the native download link. Like this: 
```html
<a :href="download.url" :download="download.name">Download</a>
```

Reply:
Updated! Nice improvement